src/main/play-doc/developer/CreatingBundles.md

```diff
@@ -84,23 +84,23 @@ Typesafe config provides the ability to substitute environment variables if they
 
 
 ```scala
-BundleKeys.endpoints := Map("customer-service" -> Endpoint("http", services = Set(URI("http://:5444/customers"))))
+BundleKeys.endpoints := Map("customer_service" -> Endpoint("http", 0, "customers", RequestAcl(Http("^/customers".r))))
```

Comment:
sorry @fsat, I must disagree. I feel the matching of service name to service specific env vars is emphasized this way.